src/main/java/org/broadinstitute/hellbender/utils/read/Fragment.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.util.Locatable;
+import org.apache.commons.lang3.tuple.Pair;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * All available evidence coming from a single biological fragment.  Either one read or a read pair.
+ */
+public class Fragment implements Locatable {
```

Comment:
Where is Fragment being used?

Reply:
Now that I apply your suggestion below, it is used.